webapi/templates/ticket-search-result.html

```diff
@@ -141,6 +143,40 @@ <h1>Vote Choices</h1>
             </tr>
         </table>
 
+        <h1>Alternate Signing Address Data</h1>
```

Comment:
```suggestion
        <h1>Alternate Signing Address</h1>
```

Don't need the word "Data" here.